pyproject.toml

```
@@ -14,8 +14,8 @@ classifiers = [
     "Programming Language :: Python :: 3",
     "Operating System :: OS Independent"
 ]
-dynamic = ["version", "dependencies"]
-requires-python = ">=3.8"
+dynamic = ["version", "dependencies", "optional-dependencies"]
+requires-python = ">=3.10"
```

I think, according to a check in `__init__` and config below.
```suggestion
requires-python = ">=3.11"
```